css-positioning/absolute.css

```diff
@@ -0,0 +1,45 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+  position: absolute;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: relative;
+  bottom: 420px;
+  left: 240px;
```

Comment:
You want to use as small offset values as possible. Using `top` & `right` would have been better here